packages/inter-protocol/src/stakeFactory/stakeFactoryKit.js

```diff
@@ -331,10 +332,6 @@ const helperBehavior = {
     const { debtBrand, manager, vaultSeat, zcf } = state;
     const { helper, pot } = facets;
     assert(state.open);
-    assertProposalShape(seat, {
```

Comment:
So the TODO above is critical; we shouldn't land this delete without also landing the pattern.